src/libraries/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -4940,6 +4945,22 @@ public enum SveMaskPattern : byte
         LargestMultipleOf3 = 30,  // The largest multiple of 3.
         All  = 31                 // All available (implicitly a multiple of two).
     };
+
+    public enum SvePrefetchType : byte
```

Comment:
@kunalspathak @tannergooding was this enum type approved? I ask because of the `SV_PLD1KEEP` names.

Reply:
https://github.com/dotnet/runtime/issues/97831 approved the `Prefetch*` APIs.

The enum should have been approved as part of that, but wasn't looked at directly.

Reply:
The enum names are different and were approved in https://github.com/dotnet/runtime/issues/94007#issuecomment-1922024205

Reply:
@TIHan  - can you update the names accordingly?